### ISSUE TITLE
Code Release

### ISSUE THREAD

**Issue by zhunzhong07** [shjung13/Standardized-max-logits#1]
Hi, thanks for your great work! When will you plan to release the code? Thanks!

**Comment by shjung13** [comment]
Hello, we are so sorry for the late update. Recently being busy with submissions. Now code is released!
